tests/garage/tf/policies/test_continuous_mlp_policy_with_model.py

```diff
@@ -0,0 +1,34 @@
+"""This script creates a unittest that tests qf-derived policy."""
+import pickle
+
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.envs import TfEnv
+from garage.tf.policies import ContinuousMLPPolicyWithModel
+from tests.fixtures import TfGraphTestCase
+from tests.fixtures.envs.dummy import DummyBoxEnv
+
+
+class TestContinuousMLPPolicyWithModel(TfGraphTestCase):
+    def setUp(self):
+        super().setUp()
+        self.env = TfEnv(DummyBoxEnv())
+        self.policy = ContinuousMLPPolicyWithModel(env_spec=self.env.spec)
+        self.env.reset()
+
+    def test_continuous_mlp_policy_with_model_get_action(self):
+        obs, _, _, _ = self.env.step(1)
```

Comment:
make sure this doesn't return an obs of all zeros, or your tests are meaningless